libfwupdplugin/fu-udev-device.c

```diff
@@ -1367,6 +1367,25 @@ fu_udev_device_get_parent_subsystems(FuUdevDevice *self)
 }
 #endif
 
+/* private */
+gboolean
+fu_udev_device_match_kind(FuUdevDevice *self, const gchar *kind)
```

Comment:
Shouldn't this be match subsystem?

Reply:
We called it "kind" in fu-device-get-backend-parent -- I figured it should be the same name. Better names welcome.

Reply:
Other ideas would be `subsys_kind`, `dev_kind`, `devsubsys`, `devtype`, `devkind`, or maybe just `subsystem`.

Reply:
It's a private function, I lean on subsystem.

Reply:
Done!